changelog.md

```diff
@@ -5,6 +5,17 @@
 **Added:**
 - New slider component, see http://backpack.prod.aws.skyscnr.com/components/web/sliders
 
+**Changed:**
+- bpk-component-modal:
+    - Renamed `bpk-modal-scrim` to `bpk-scrim`, and `BpkModalScrim` to `BpkScrim`
+- bpk-tokens and bpk-mixins:
```

Comment:
We haven't yet deprecated these yet - we've simply marked them for deprecation. At some point in the future we'll do a sweep of all the deprecated tokens and release a MAJOR version of `bpk-tokens` etc.

Essentially I dont think you need a changelog entry for this PR at all 😇 .

Reply:
Removed